cmd/opentelemetry/app/exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafkaexporter
+
+import (
+	"context"
+
+	"github.com/spf13/viper"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/config/configtls"
+	"go.opentelemetry.io/collector/exporter/kafkaexporter"
+
+	"github.com/jaegertracing/jaeger/cmd/opentelemetry/app/receiver/kafkareceiver"
+	"github.com/jaegertracing/jaeger/plugin/storage/kafka"
+)
+
+// TypeStr defines exporter type.
+const TypeStr = "kafka"
```

Comment:
Where is this being used?

Reply:
https://github.com/jaegertracing/jaeger/pull/2494/files#diff-049d063f5338468c88d1c32ad60055efR83-R90